src/feature/client/entrynodes.h

```diff
@@ -116,6 +116,12 @@ struct entry_guard_t {
    * successfully and decide to keep it?) This field is zero if this is not a
    * confirmed guard. */
   time_t confirmed_on_date; /* 0 if not confirmed */
+  /**
+   * In what order was this guard sampled without replacement? Guards with
+   * lower indices appear earlier on the sampled list
+   */
```

Comment:
I think we need a patch for `guard-spec.txt` to explain how this new field should work. Our guard spec is quite engineering oriented and basically explains how these fields tie together, so I think mentioning `sampled_idx` and `next_sampled_idx` there is important.

Reply:
I can send a PR to the torspec repository (I guess guard-spec.txt is there)

Reply:
Yes please!